pkg/crds/yaml/generated/upgrade.cattle.io_plans.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.17.3
+    controller-gen.kubebuilder.io/version: v0.18.0
```

Comment:
looks like you're using a different version of kube-builder than our CI. Would you mind reverting this bit?